casper/src/test/scala/coop/rchain/casper/genesis/contracts/TestUtil.scala

```diff
@@ -77,19 +74,21 @@ object TestUtil {
 
   private def evalDeploy[F[_]: Sync](
       deploy: DeployData,
-      runtime: Runtime[F]
+      runtime: Runtime[F],
+      deployerPk: Option[PublicKey]
```

Comment:
There's no need for `deployerPk` argument, and in fact there's a need for there to not be one :) It should be taken from `deploy: DeployData`

Reply:
Done